memberships/templates/base.html

```diff
@@ -13,44 +17,59 @@
   <body onload="checkDarkMode()">
     <div class="header-container">
       <header>
-        <h1><a href="/" title="Home">Geek.Zone</a></h1>
-        
-        <nav>
-          <ul>
+
+
+        <nav class="navbar navbar-expand-lg">
+          <h1><a href="/" title="Home">Geek.Zone</a></h1>
+          <button class="navbar-toggler navbar-toggle" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <div class="">
+              <div class="line"></div>
+              <div class="line"></div>
+              <div class="line"></div>
+            </div>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav">
             {% if user.is_authenticated %}
-            <li>
-              <a href="{% url 'memberships_details' %}">Membership Details</a>
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'memberships_details' %}">Membership Details</a>
             </li>
-            
-            <li>
-              <a href="{% url 'memberships_logout' %}">Logout</a>
+
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'memberships_logout' %}">Logout</a>
             </li>
             {% else %}
-            <li>
-              <a href="{% url 'register' %}">Register</a>
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'register' %}">Register</a>
             </li>
-            
-            <li>
-              <a href="{% url 'memberships_login' %}">Login</a>
+
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'memberships_login' %}">Login</a>
             </li>
             {% endif %}
-            
-            <li>
-              <a href="https://geek.zone/tiki-searchindex.php" target="_blank">Wiki</a>
+
+            <li class="nav-item">
+              <a class="nav-link" href="https://geek.zone/tiki-searchindex.php" target="_blank">Wiki</a>
             </li>
           </ul>
+          </div>
         </nav>
       </header>
     </div>
-    
+
     <main>
       {% block content %}
       {% endblock %}
     </main>
-    
+
     <footer>
       <h3><span style="display: inline-block; transform: rotateY(180deg);">&copy;</span> Geek.Zone</h3>
       <div><button class="button" onclick="toggleDarkMode()">Toggle Dark Mode</button></div>
     </footer>
+    <!-- Optional JavaScript -->
+  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
We are trying to avoid using jQuery where possible as the vast majority of its features are built into browsers now.

Thankfully bootstrap 5, which is in beta so good enough, has removed their jquery dependency.

I'll go ahead and merge this anyway for now, and I'll raise a PR to upgrade us to bootstrap 5.

I don't believe they have changed the any significant visual elements.